app/routes/application.js

```diff
@@ -4,6 +4,12 @@ import config from '../config/environment';
 
 export default Ember.Route.extend({
   beforeModel: function (transition) {
+    //for language for ggv_orders from url
+    if(transition.targetName === "ggv_orders") {
+      var language = transition.queryParams.ln === "zh-tw" ? "zh-tw" : "en";
+      this.set('session.language', language);
+    }
+
```

Comment:
Sure @steveyken, I will change it. I thought, we should not allow user to switch language from url other than ggv_orders as we have langugae switcher on home page.

Thanks